docs/overview.yaml

```diff
@@ -245,6 +246,7 @@ Analysis:
     Fréchet-Riesz representation of the dual of a Hilbert space: 'inner_product_space.to_dual'
     isometric inclusion in double dual: 'normed_space.inclusion_in_double_dual_isometry'
     completeness of spaces of bounded continuous functions: 'bounded_continuous_function.complete_space'
+  Transcendental numbers:
```

Comment:
Shouldn't we move this line to 'Number theory' as well?
```suggestion
```

Reply:
I just did!